internal/ingress/controller/template/template.go

```diff
@@ -602,6 +636,76 @@ func buildAuthProxySetHeaders(headers map[string]string) []string {
 	return res
 }
 
+func buildAuthUpstreamName(input interface{}, host string) string {
+	authPath := buildAuthLocation(input, "")
+	if authPath == "" || host == "" {
+		return ""
+	}
+
+	return fmt.Sprintf("%v-%v", host, authPath[2:])
```

Comment:
and here :) let's not stick with the interface generic value, but try to always use the right modifier (%d for int, %s for string, etc)